### PR TITLE
Adds the FacebookSiteName component

### LINKED ISSUE #807: Create a FacebookSiteName component
The `FacebookSiteName` will be shown as a div:
yoast_-_home

The sitename only contain the 'domain' part. The scheme and the path should be stripped out. 

## Styling
The styling for this will be:
```css
    color: #606770;
    font-size: 12px;
    line-height: 11px;
    text-transform: uppercase;
    overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
}
```

Parent #811

### PR DESCRIPTION
**Pull request by andizer**
## Summary

This PR can be summarized in the following changelog entry:

* Adds a `FacebookSiteName` components

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Adds this to the App.js file:
```js

	{
		id: "sitename-example",
		name: "FacebookSiteName",
		component: <FacebookSiteName siteName="Yoast.com" />,
	},
```
* Do `yarn start`
* And view the component in the example.

## UI changes
* [ ] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

Fixes #807

### CODE REVIEW

**Comment by IreneStr** [comment]
CR 👍Please add a new Facebook Preview tab to the stand-alone example, and add the code from the test instructions there.

File: composites/Plugin/SocialPreviews/Facebook/components/FacebookPreview.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import FacebookSiteName from "./FacebookSiteName";
+
+
+/**
+ * Renders a FacebookSite preview component.
```
Comment (IreneStr) [inline composites/Plugin/SocialPreviews/Facebook/components/FacebookPreview.js]:
```suggestion
 * Renders a FacebookPreview component.
```

File: composites/Plugin/SocialPreviews/Facebook/components/FacebookPreview.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * @param {object} props The props.
+ *
+ * @returns {React.Element} The rendered element.
+ *
+ * @constructor
+ */
+const FacebookPreview = ( props ) => {
+	return (
+		<FacebookSiteName siteName={  props.siteName } />
```
Comment (IreneStr) [inline composites/Plugin/SocialPreviews/Facebook/components/FacebookPreview.js]:
```suggestion
		<FacebookSiteName siteName={ props.siteName } />
```

**Comment by IreneStr** [comment]
CR done 👍 Please apply the 2 suggestions

**Comment by Dieterrr** [comment]
Acceptance done 👍 

Technically there is a tiny UI change from the old preview to the new: The font size has changed from 11 to 12 and the color has changed #9197a3. It is barely noticeable.

**Pull request merged**